storage/backend/null/null.go

```diff
@@ -2,10 +2,20 @@
 
```

Comment:
Yes, we need to incorporate them because they need to be finished, and @mapuri is already committed to doing this work.